procedures/katello/release.md.erb

```diff
@@ -58,6 +59,7 @@
 
 - [ ] Confirm response that the build succeeded (or if necessary, do more cherry-picks and version bumps repeating the steps above)
 - [ ] Post a [release announcement](https://community.theforeman.org/c/release-announcements/8)
-<% unless is_rc %>
+<% if !is_rc && full_version[-1] == '0' %>
```

Comment:
Small nit: I like to be precise. We rarely get to `.10`, but it's good to think about. Also `end_with?` is easier to read IMHO.
```suggestion
<% if !is_rc && full_version.end_with?('.0') %>
```